### PR TITLE
Changelog 2.5

### PR DESCRIPTION
**Pull request by mauriciopoppe**
**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

/cc @jingxu97 @xing-yang

### CODE REVIEW

File: CHANGELOG/CHANGELOG-2.5.md [hunk 1]
```diff
@@ -0,0 +1,98 @@
+## Changes by Kind
```
Comment (msau42) [inline CHANGELOG/CHANGELOG-2.5.md:1]: Oops I missed this. Can you also add the headers with the release number?

Comment (mauriciopoppe) [reply CHANGELOG/CHANGELOG-2.5.md:1]: sure, I didn't notice that, thanks

**Pull request merged**